.goreleaser

```diff
@@ -0,0 +1,51 @@
+---
+project_name: terraform-provider-spinnaker
+builds:
+  - main: ./main.go
+    ldflags:
+    - '-s -w -X main.version={{.Version}} -X main.commit={{.Commit}}'
+    env:
+      - CGO_ENABLED=0
+    goos:
+      - linux
+      - darwin
+    goarch:
+      - 386
+      - amd64
+    binary: '{{ .ProjectName }}_v{{ .Version }}'
+archives:
+  - format: binary
+    files:
+      - none*
+    replacements:
+      386: i386
+      amd64: x86_64
+  - format: zip
+  name_template: '{{ .ProjectName }}_{{ .Version }}_{{ .Os }}_{{ .Arch }}'
+checksum:
+  name_template: '{{ .ProjectName }}_{{ .Version }}_SHA256SUMS'
+  algorithm: sha256
+signs:
+  - artifacts: checksum
+    args:
+      - "--batch"
+      - "--local-user"
+      - "{{ .Env.GPG_FINGERPRINT }}" # set this environment variable for your signing key
```

Comment:
I couldn't find this variable in 
https://github.com/crazy-max/ghaction-import-gpg

therefore  I think we should keep in our mind that we must add this in github variables